### PR TITLE
#88: Add some files to configure the project

### PR DESCRIPTION
**Pull request by benedeki**
* Git configuration
* Scalastyle support

### CODE REVIEW

**Comment by benedeki** [comment]
> 
> 
> please add scala style pom.xml entry. Enceladus's entry https://github.com/AbsaOSS/enceladus/blob/develop/pom.xml#L344
> and maybe a GitHub action for it?
> 
> Otherwise looks good

Of course. Silly me to forget that. But what GitHub action you have in mind?

**Comment by Zejnilovic** [comment]
> > please add scala style pom.xml entry. Enceladus's entry https://github.com/AbsaOSS/enceladus/blob/develop/pom.xml#L344
> > and maybe a GitHub action for it?
> > Otherwise looks good
> 
> Of course. Silly me to forget `pom.xml` entry. But what GitHub action you have in mind?

Well, does it run always on Build? Or it needs to be run separately?

**Comment by benedeki** [comment]
> 
> 
> > > please add scala style pom.xml entry. Enceladus's entry https://github.com/AbsaOSS/enceladus/blob/develop/pom.xml#L344
> > > and maybe a GitHub action for it?
> > > Otherwise looks good
> > 
> > 
> > Of course. Silly me to forget `pom.xml` entry. But what GitHub action you have in mind?
> 
> Well, does it run always on Build? Or it needs to be run separately?

Ah, part of build.

**Comment by benedeki** [comment]
Oops, it grew a little. I promise I won't add more to it 😄

**Review by dk1844 — APPROVED** [review]
LGTM, thanks for adding the cross-build for Scala2.12/Spark3.1 and also providing the CMD version of the script

(just read the code)

**Review by Zejnilovic — APPROVED** [review]

**Review by AdrianOlosutean — DISMISSED** [review]

**Comment by benedeki** [comment]
Changed the `build-all.sh` and `build-all.cmd scripts to stop on fail.
Local build succeeded and this time even the server build, as I see. 🤔

**Review by Zejnilovic — APPROVED** [review]

File: build-all.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #
 
 set -e
-
+build.yml
```
Comment (Zejnilovic) [inline build-all.sh:22]: why?

Comment (benedeki) [reply build-all.sh:22]: By mistake.

**Review by Zejnilovic — APPROVED** [review]

**Review by dk1844 — APPROVED** [review]

**Pull request merged**